content/en/getting_started/synthetics/api_test.md

```diff
@@ -37,34 +40,41 @@ The example below demonstrates the creation of an [HTTP test][2], a subtype of [
     - You can select **Advanced Options** to use custom request headers, authentication credentials, body content, or cookies.
     - You can set tags such as `env:prod` and `app:shopist` on your test. Tags allow you to keep your test suite organized and quickly find tests you're interested in on the homepage.
+    - You can use secure [global variables][11] for any credentials in your API call. You can also create [local variables][12] to inject a dynamically defined timestamp in your request payload. After creating these variables, type `{{` and select a variable to inject the variable in your test options.
 
 6. Click **Test URL** to trigger a sample test run.
 
-{{< img src="getting_started/synthetics/api-test-config-2.jpg" alt="API test configuration"  style="width:60%;">}}
+{{< img src="getting_started/synthetics/api-test-config-3.png" alt="API test configuration"  style="width:100%;">}}
 
 ### Define assertions
 
 Clicking **Test URL** automatically populates basic assertions about your endpoint's response. Assertions define what a successful test run is.
 
 In this example, three default assertions populate after triggering the sample test run:
 
-{{< img src="getting_started/synthetics/assertions-example.png" alt="Browser test failure"  style="width:90%;">}}
+{{< img src="getting_started/synthetics/assertions-example-2.png" alt="Default assertions"  style="width:100%;">}}
 
 Assertions are fully customizable. To add a custom assertion, click on elements of the response preview or click **New Assertion**. 
 
-{{< img src="getting_started/synthetics/api-test-configuration-2.gif" alt="Browser test failure"  style="width:90%;">}}
+{{< img src="getting_started/synthetics/api-test-configuration-2.mp4" alt="Example API test configuration"  video="true"  >}}
+
+**Note**: You can also leverage [global][5] and [local][6] variables in your assertions.
 
 ### Select locations 
 
 Select one or more **Managed Locations** or **Private Locations** to run your test from.
 
 Managed locations allow you to test public-facing websites and endpoints. To test internal applications or simulate user behavior in discrete geographic regions, select one of your **Private Locations** instead.
 
-For more information on how to set up private locations, see [Getting Started with Private Locations][5].
+For more information on how to set up private locations, see [Getting Started with Private Locations][7].
 
 ### Specify test frequency
 
-Select the frequency of your test runs.
+Select the frequency at which you want your test to execute. 
+
+In addition to running your Synthetic test on a schedule, you can trigger them directly from your CI/CD pipelines. For more information, see [Synthetic CI/CD Testing][8].
+
+For more information, see [Synthetic CI/CD Testing][8].
```

Comment:
This appears twice, let's remove that one!

Reply:
```suggestion
```